config/durability/near_full_data_storage.yaml

```diff
@@ -0,0 +1,3 @@
+full_sys_writes:
```

Comment:
Can we rather add a config file which is common for all durability tests? The name can be general enough like test_durability.yaml maybe
And may not require to create under a new dir , it can be under config itself